examples/kubernetes/modify-volume/README.md

```diff
@@ -2,7 +2,7 @@
 
 ## Prerequisites
 
-This example will only work on a cluster with the `VolumeAttributesClass` feature enabled. For more information see the [installation instructions in the EBS CSI `ModifyVolume` documentation](/docs/modify-volume.md).
+This example will only work with `aws-ebs-csi-driver` >= v1.35.0 on a Kubernetes v1.31 cluster with the beta `VolumeAttributesClass` feature enabled. For more information see the [installation instructions in the EBS CSI `ModifyVolume` documentation](/docs/modify-volume.md).
```

Comment:
Drop "with `aws-ebs-csi-driver` >= v1.35.0" - docs on `master` refer to the latest version of the driver so this is redundant.

Also, "Kubernetes `v1.31`" should be "Kubernetes `v1.31` or later"

```suggestion
This example will only work on a Kubernetes v1.31 or later cluster with the beta `VolumeAttributesClass` feature enabled. For more information see the [installation instructions in the EBS CSI `ModifyVolume` documentation](/docs/modify-volume.md).
```